src/app/component/allresults/allresults.component.css

```diff
@@ -0,0 +1,70 @@
+body {
```

Comment:
why to direct  body ?

Reply:
Please use scss format style

Reply:
yes i used scss format file and removed body css